tests/libvmi/cloudinit.go

```diff
@@ -22,14 +22,13 @@ package libvmi
 import (
 	"encoding/base64"
 
-	kvirtv1 "kubevirt.io/api/core/v1"
 	v1 "kubevirt.io/api/core/v1"
```

Comment:
Could we define a single way to shorten the kubevirt packages in the code? I know that this is independent of this PR, but we have at least 3 ways in the code `kvirtv1`, `k6*` and sometimes just `v1` as in this case. WDYT?

Reply:
No, there isn't such a written rule, I'm just simply asking here because you choose to use `v1` instead of `kvirtv1`

Reply:
Well, I used the logic I mentioned earlier and looked for usages of `VirtualMachineInstance` which is widely used in the tests. It showed that in most cases it uses `v1`.
Nothing beyond that.